vpr/test/test_connection_router.cpp

```diff
@@ -0,0 +1,140 @@
+#include "catch.hpp"
```

Comment:
This needs comments.

Should explain what the test is testing and why it exists.

Reply:
Done.